moq-arch.md

```diff
@@ -619,10 +619,10 @@ may have to be realtime.
 
 The basic idea in BBR of speeding up to probe then slowing down to drain
 the queue build up caused during probe can work fine with real time
-applications. However the current implementations in QUIC do not
-seem optimized for real time applications and have some times where the
-slow down causes too much jitter. To not have playout drops, the jitter
-buffers adds latency to compensate for this. Probing for the RTT has been
+applications. However, the current implementations in QUIC do not
+appear optimized for real-time applications, resulting in higher jitter.
```

Comment:
we need to qualify higher jitter under certain conditions .. with all good network, it doesn't lead to the issues. May be we can rewrite it as 
s/resulting in higher jitter/resulting in higher jitter (under certain conditions)